### PR TITLE
bugfixes

### PR DESCRIPTION
**Pull request by ajay-arora**
Fixes for bugids : 419898, 419899, 419900, 419911, 419924
Add/Update readmes for Ant, Maven, Gradle, Vstest (place holders)
Updating patch version
Updated Coverage option name
Updated missing tooltips for options/target fields

Testing : Uploaded the new tasks and verified the changes in the UI. No functional coverage

### CODE REVIEW

File: Tasks/ANTPreview/task.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
                   ],    
     "author": "Microsoft Corporation",
     "version": {
-        "Major": 0,
+        "Major": 1,
```
Comment (AshwiniChalla) [inline Tasks/ANTPreview/task.json]: For my understanding. When do we need major version change?

**Comment by AshwiniChalla** [comment]
Please specify the testing

File: Tasks/ANTPreview/task.json (hunk 1 above)
Comment (ajay-arora) [inline Tasks/ANTPreview/task.json]: check description ... the version should start from 1. We were starting from 0, so i fixed that.

**Comment by ajay-arora** [comment]
reviewed with Atin. Checking in

**Pull request merged**